lib/roo_on_rails/railties/http.rb

```diff
@@ -28,10 +28,13 @@ class HTTP < Rails::Railtie
           app.config.middleware.use ::Rack::Deflater
         end
 
-        app.config.middleware.insert_before(
-          ActionDispatch::Cookies,
-          ::Rack::SslEnforcer
-        )
+        # Don't use SslEnforcer in test environment as it breaks Capybara
+        unless Rails.env.test?
+          app.config.middleware.insert_before(
+            ActionDispatch::Cookies,
```

Comment:
How do you manage it in logistics-dashboard - I assume IP based? i.e. I can go to https://logistics-dashboard-production.herokuapp.com/sidekiq without any auth step, but I need to use google auth to go to https://logistics-dashboard-production.herokuapp.com/. My understanding is that the sidekiq dashboard is completely separate from whatever you do in rails.